packages/server/src/__tests__/documentStore.test.ts

```diff
@@ -46,28 +48,20 @@ describe('ApolloServerBase documentStore', () => {
 
     // Use [] syntax to access a private method.
     const { documentStore } = await server['_ensureStarted']();
-    expect(documentStore).toBeInstanceOf(InMemoryLRUCache);
-    const embeddedStore = documentStore as InMemoryLRUCache<DocumentNode>;
+    assert(documentStore);
```

Comment:
Neat — wonder if we should use it instead of `expect(...).toBeTruthy()` in general...

Reply:
I think so. The only rub for testing maybe is that it throws instead of wraps the throw in an `expect(() => { ... }).toThrow()` but the end result is a test failure so maybe it's fine? 🤷 

Or, lol:
`expect(() => assert(xyz)).not.toThrow()`